packages/angular/ssr/src/app.ts

```diff
@@ -51,9 +51,21 @@ const SERVER_CONTEXT_VALUE: Record<RenderMode, string> = {
  * The `AngularServerApp` class handles server-side rendering and asset management for a specific locale.
  */
 export class AngularServerApp {
+  /**
+   * A flag to enable or disable the rendering of prerendered routes.
+   *
+   * Typically used during development to avoid prerendering all routes ahead of time,
+   * allowing them to be rendered on the fly as requested.
+   *
+   * @private
+   */
+  allowStaticRouteRender = false;
```

Comment:
Consider: Could we make this a `readonly` constructor parameter? We probably wouldn't want this to change mid execution. Maybe it could also be directly marked `private` instead of needing `@private`?